### DISCUSSION TITLE
clef.exe import exported loggins - traceid missing and no trace timings

### DISCUSSION THREAD

**Discussion by TomDC-DX** [datalust/seq-tickets#2329] [Help] [answered]
Hello,

I'm looking at the SEQ import functionality by using CLEF files.

I exported one day of logging in two ways:
- using the website export functionality
- using serilog clef file logging

Both ways are configured like documented on the SEQ website.

However, upon import using the latest clef.exe, I notice that a lot of information is missing after import.
I can no longer filter on traceid and all the additional timing information is also missing.

I see these new columns after import, which were not there before:

![image](https://github.com/user-attachments/assets/d3627b4e-e8bb-45b4-8155-cd5cc9a8ab3e)

Did I do anything wrong?
Any help is usefull, since we are looking to implement this on production servers and the CLEF file should be the "backup log".

**UPDATE!**

I did some searching and debugging and I noticed the following.

When importing from a clef file, it seems that something goes wrong with the @sp and @tr tags?

![image](https://github.com/user-attachments/assets/9bade5ee-c305-4d2c-8712-1fca077d8b51)

Thanks!

**Comment by nblumhardt** [comment, marked as answer]
Hi @TomDC-DX,

`clef.exe` is in maintenance mode and hasn't been updated to support these newer properties. Instead, try `seqcli ingest --json -i <FILENAME>` using a recent version of `seqcli`.

On Windows, `seqcli` should already be on your path if you have Seq installed on Windows; it's also available as a `dotnet tool`, binaries, or as a Docker container. Check out https://github.com/datalust/seqcli for all the info.

Hope this helps,
Nick

**Reply by TomDC-DX** [reply]
Many thanks!

The seqcli command can restore everything, including the tracings, using the CLEF files.
